README.md

```diff
@@ -154,8 +154,8 @@ This getting started guide will use the most fundamental concepts of OpenAPI and
 
 1. Create path handlers.
 
-    Our `paths` object was empty in the main apiDoc because `express-openapi` generates
-    it for us based on the location of our path handlers.  For this example we'll place
+    Our `paths` object was empty in the main apiDoc because `express-openapi` allows to
```

Comment:
Can you change this to:

```
Our `paths` object was empty in the main apiDoc because `express-openapi` will generate it for us based on the location of our path handlers.
```